test/Optimizer/rlexe.xml

```diff
@@ -1560,6 +1560,12 @@
       <compile-flags> -loopinterpretcount:1 -bgjit- -maxsimplejitruncount:1 -maxinterpretcount:1 -oopjit-</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bugsimplepathbrfold5.js</files>
+      <tags>exclude_dynapogo,exclude_nonative</tags>
+    </default>
```

Comment:
To make this repro for me I had to add `-mic:1 -off:simplejit` OR run it with dynamic profile info.